### PR TITLE
add new configuration and authentication section https://github.com/g…

### LINKED ISSUE #769: Major version control authentication support for SVN
**Is your feature request related to a problem? Please describe.**
Lack of setup for SVN authentication support.

**Describe the solution you'd like**
Create authentication support for SVN in the proxy.

### PR DESCRIPTION
**Pull request by gcstang**
…omods/athens/issues/769

**What is the problem I am trying to address?**
https://github.com/gomods/athens/issues/769

**How is the fix applied?**

I created a new directory under docs/content/configuration along with an index and authentication md file.  The authentication file currently has just the SVN setup for authentication but as I mentioned in the issue 769 there is a file also under docs/content/install/ named 'managing-private-repos.md' that could be moved.

**Mention the issue number it fixes or add the details of the changes if it doesn't has a specific issue.**

Fixes #769

### CODE REVIEW

File: docs/content/configuration/authentication.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+## Authentication
```
Comment (robjloranger) [inline docs/content/configuration/authentication.md]: You need front matter here to register it as a page.

Check out another non index page for an example.

File: docs/content/configuration/authentication.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+md -Path $env:ATHENS_STORAGE
+docker run -d -v "$($env:ATHENS-STORAGE):/var/lib/athens" `
+   -v "$($env:ATHENS-SVN):/root/.subversion" `
+   -e ATHENS_DISK_STORAGE_ROOT=/var/lib/athens `
+   -e ATHENS_STORAGE_TYPE=disk `
+   --name athens-proxy `
+   --restart always `
+   -p 3000:3000 `
+   gomods/proxy:latest
+```
```
Comment (robjloranger) [inline docs/content/configuration/authentication.md]: Here's where that new line is missing.

**Comment by robjloranger** [comment]
This is looking really nice.

I'll wait for others to review, and when I get a chance I will verify the technical content myself as well.

Thanks so much for this @gcstang

File: docs/content/configuration/authentication.md (hunk 2 above)
Comment (gcstang) [reply docs/content/configuration/authentication.md]: added to end of file

File: docs/content/configuration/_index.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+weight: 3
+---
+
+## Configuring Athens
+Here we'll cover how to configure the Athens application utilizing various configuration scenarios.
+
+### Authentication
+There are numerous version control systems available to us as developers.  In this section we'll outline how they can be used by supplying required credentials in various formats for the Athens project.
+
+ - [Authentication](https://github.com/gomods/athens/blob/master/docs/content/configuration/authentication.md)
```
Comment (arschles) [inline docs/content/configuration/_index.md]: @gcstang looks like this file is missing. did you mean something on the docs.gomods.io site?

Comment (gcstang) [reply docs/content/configuration/_index.md]: it's part of this PR

Comment (arschles) [reply docs/content/configuration/_index.md]: doh, sorry!

**Comment by arschles** [comment]
Thanks again for this @gcstang . merging now 😄

**Pull request merged**